tests/integration/local_toolchains/.bazelrc

```diff
@@ -0,0 +1,5 @@
+common --action_env=RULES_PYTHON_BZLMOD_DEBUG=1
```

Comment:
Oh, using this in tests is really useful. Should we have this in CI for the `examples` jobs as well?

Reply:
I think having a `Troubleshooting` section might be good. We could point to Troubleshooting sub-sections within Toolchains, PyPI integration, etc.